sklearn/impute/_base.py

```diff
@@ -1,7 +1,3 @@
-# Authors: Nicolas Tresegnie <nicolas.tresegnie@gmail.com>
```

Comment:
Why did you remove the author information?

Reply:
I didn't do that intentionally, its a mistake, sorry I'll restore it immediately.

Reply:
Wait a minute, its still there in my local copy.

Reply:
See,
![Screen Shot 2019-07-14 at 10 22 34 PM](https://user-images.githubusercontent.com/39561084/61186702-f2731400-a685-11e9-816d-bec511652877.png)

I don't know why the diff shows so.